status-payloads-spec.md

```diff
@@ -134,94 +192,169 @@ All incoming messages can be matched against a chat. Below you can find a table
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the contact synced |
+| 3 | profile_image | `string` |  `base64` encoded profile picture of the user |
+| 4 | ens_name | `string` | ENS name of the contact |
+| 5 | `array[string]` | Array of `system_tags` for the user, this can currently be: `":contact/added", ":contact/blocked", ":contact/request-received"`|
 
-### SyncInstallation
+### SyncInstallationPublicChat
 
-`SyncInstallation` messages are used to synchronize in a best-effort way all the paired installations. It is identified by a transit tag of `p1` 
+`SyncInstallationPublicChat` message is used to synchronize in a best-effort the public chats to other devices.
+
+```protobuf
+message SyncInstallationPublicChat {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
 
 #### Payload
 
 | Field | Name | Type | Description |
 | ----- | ---- | ---- | ---- |
-| 1| contacts | `[struct { name string last-updated int device-info struct {id string fcm-token string } pending? bool}` | An array of contacts |
-| 2 | account | `struct {name string photo-path string last-updated int}` | Information about your own account |
-| 3 | chat | `struct {:public? bool :chat-id string}` | A description of a public chat opened by the client |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the chat synced |
 
 ### PairInstallation
 
-`PairInstallation` messages are used to propagate informations about a device to its paired devices. It is identified by a transit tag of `p2` 
+`PairInstallation` messages are used to propagate informations about a device to its paired devices.
+
+```protobuf
+message PairInstallation {
+  uint64 clock = 1;
+  string installation_id = 2;
+  string device_type = 3;
+  string name = 4;
+}
+```
 
 #### Payload
 
 | Field | Name | Type | Description |
 | ----- | ---- | ---- | ---- |
-| 1| installation-id | `string` | A randomly generated id that identifies this device |
-| 2 | device-type | `string` | The OS of the device `ios`,`android` or `desktop` |
-| 3 | name | `string` | The self-assigned name of the device |
-| 4 | fcm-token | `string` | The FCM Token used by mobile devices for push notifications |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | installation_id | `string` | A randomly generated id that identifies this device |
+| 3 | device_type | `string` | The OS of the device `ios`,`android` or `desktop` |
+| 4 | name | `string` | The self-assigned name of the device |
 
-### GroupMembershipUpdate
+### MembershipUpdateMessage and MembershipUpdateEvent
 
-`GroupMembershipUpdate` is a message used to propagate information about group membership changes in a group chat.. It is identified by a transit tag of `g5`.
+`MembershipUpdateEvent` is a message used to propagate information about group membership changes in a group chat.
 The details are in the  [Group chats specs](status-group-chats-spec.md)
 
+```protobuf
+message MembershipUpdateEvent {
```

Comment:
I feel like I have seen this before, is there a reason to duplicate it?

Reply:
Moved to `status-group-chats-spec.md`